health/egov-persister/attendance-service-persister.yml

```
@@ -0,0 +1,414 @@
+
+       - jsonPath: $.attendance[*][?({id} in @.documentIds[*].id)].auditDetails.lastModifiedBy
+
+       - jsonPath: $.attendance[*][?({id} in @.documentIds[*].id)].auditDetails.createdTime
+
+       - jsonPath: $.attendance[*][?({id} in @.documentIds[*].id)].auditDetails.lastModifiedTime
+       
+       - jsonPath: $.attendance[*][?({id} in @.documentIds[*].id)].tenantId
+
+       - jsonPath: $.attendance.*.documentIds.*.status
+
+ 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)